From 4e237478f3654c07f2cee76d0e7295029000e681 Mon Sep 17 00:00:00 2001
From: Lionel Debieve <lionel.debieve@foss.st.com>
Date: Wed, 8 Dec 2021 17:36:03 +0100
Subject: [PATCH 0334/1695] plat-stm32mp1: conf: enable EXTI driver for
 STM32MP13

Add the EXTI management for STM32MP13 platform.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 7c4864b24843bc2a9220c3a1774e06cda57786d0 plat-stm32mp1: conf: enable EXTI driver for STM32MP13]
Signed-off-by: Lionel Debieve <lionel.debieve@foss.st.com>
Change-Id: Idba120046d5a358f2a42ac2f118287e6534e0217
---
 core/arch/arm/plat-stm32mp1/conf.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/plat-stm32mp1/conf.mk b/core/arch/arm/plat-stm32mp1/conf.mk
index 46be4a336..66ba29cc5 100644
--- a/core/arch/arm/plat-stm32mp1/conf.mk
+++ b/core/arch/arm/plat-stm32mp1/conf.mk
@@ -160,6 +160,7 @@ $(call force,CFG_CORE_ASYNC_NOTIF_GIC_INTID,31)
 $(call force,CFG_CORE_RESERVED_SHM,n)
 $(call force,CFG_DRIVERS_CLK_FIXED,y)
 $(call force,CFG_SECONDARY_INIT_CNTFRQ,n)
+$(call force,CFG_STM32_EXTI,y)
 $(call force,CFG_STM32_GPIO,y)
 $(call force,CFG_STM32_VREFBUF,y)
 $(call force,CFG_STM32MP_CLK_CORE,y)
-- 
2.47.2

